docs/gitbook/chain-management/cascading-deployment.md

```diff
@@ -0,0 +1,85 @@
+# Cascading Deployment
+
+
+<div class="tips info">
+  <p><span></span>Coming soon...</p>
+  <p>The cascading release managed by a root node with a set of adming keys is not available yet.</p>
+</div>
+
+Managing an entire blockchain system that exists on many nodes with different key pairs with chains that join in myriad ways can be a headache to manage. Cascading deployment offers a way to maintain a blockchain network from a single point of change.
+
+There are several important components used to implement cascading deployment in an Interbit blockchain network.
+
+## The Interbit Configuration File
+
+The configuration file is a js file that contains intstructions on generating a collection of build artifacts.
+Contains:
+- expressions for resolving variables for:
+  - pubkeys /  chainIds (basically permission resolvers)
+  - chain Ids
+  - covenant hashes
+  - peer lists
+  - folder containing index.html and other static assets to be hosted.  index.html will get altered by the build step to include chainIds
+  - array of supporting static files that should be hosted statically
```

Comment:
JavaScript files can't contain folders. 'path to folder containing'...
'array of supporting files that should be hosted statically'